src/ui/components/Assist/Assist.tsx

```diff
@@ -0,0 +1,206 @@
+  const showInspectElement = shouldShowInspectElement(nags);
+  const showInspectComponent = shouldShowInspectComponent(nags);
+  const showUseFocusMode = shouldShowUseFocusMode(nags);
+
+  const [showConfetti, setShowConfetti] = useState(false);
+
+  const info = useTestInfo();
+
+  const renderCheckmarkIcon = (completed: boolean | undefined) => {
+    if (completed === false) {
+      return "checked-rounded";
+    }
+    return "unchecked-rounded";
+  };
+
+  const [stepIndex, setStepIndex] = useState(0);
+
+  const handleClick = (index: number) => {
+    setStepIndex(index);
+    setSelectedItem(index);
+  };
+
+  const getItemStyle = (index: number) => {
+    return selectedItem === index ? styles.selectedItem : "";
+  };
+
+  const updatedChecklistItems = [
+    {
+      label: "Time travel in the console",
+      completed: showConsoleNavigate,
+      videoUrl: "https://vercel.replay.io/assist/time_travel_in_console.gif",
+    },
+    {
+      label: "Set a print statement",
+      completed: showBreakpointEdit,
+      videoUrl: "https://vercel.replay.io/assist/set_print_statement.gif",
+    },
+    {
+      label: "Launch command palette",
+      completed: showLaunchCommandPalette,
+      videoUrl: "https://vercel.replay.io/assist/launch_command_palette.gif",
+    },
+    {
+      label: "Explore sources",
+      completed: showExploreSources,
+      videoUrl: "https://vercel.replay.io/assist/explore_sources.gif",
+    },
+    {
+      label: "Search source text",
+      completed: showSearchSourceText,
+      videoUrl: "https://vercel.replay.io/assist/search_source_text.gif",
+    },
+    {
+      label: "Jump to event",
+      completed: showJumpToEvent,
+      videoUrl: "https://vercel.replay.io/assist/jump_to_an_event.gif",
+    },
+    {
+      label: "Jump to code",
+      completed: showJumpToCode,
+      videoUrl: "https://vercel.replay.io/assist/jump_to_code.gif",
+    },
+    {
+      label: "Add a comment",
+      completed: showAddComment,
+      videoUrl: "https://vercel.replay.io/assist/add_a_comment.gif",
+    },
+    {
+      label: "Add a unicorn badge",
+      completed: showAddUnicornBadge,
+      videoUrl: "https://vercel.replay.io/assist/unicorn_badge.gif",
+    },
+    {
+      label: "Use focus mode",
+      completed: showUseFocusMode,
+      videoUrl: "https://vercel.replay.io/assist/use_focus_mode.gif",
+    },
+    {
+      label: "Record a replay",
+      completed: showRecordReplay,
+      videoUrl: "https://vercel.replay.io/assist/record_a_replay.gif",
+    },
+    {
+      label: "Inspect element",
+      completed: showInspectElement,
+      videoUrl: "https://vercel.replay.io/assist/inspect_an_element.gif",
+    },
+  ];
+
+  useEffect(() => {
+    const firstUnselectedIndex = updatedChecklistItems.findIndex(item => !item.completed);
+    if (firstUnselectedIndex !== -1) {
+      setSelectedItem(firstUnselectedIndex);
+      setStepIndex(firstUnselectedIndex);
+    }
+  }, []);
```

Comment:
nit: There's still no reason to store `selectedItem` in state.  It can be calculated from the `stepIndex` (also stored in state) and storing them separately can lead to mismatches.

It's not a big deal in this case since the component is small and it's unlikely that one value would be updated without the other, but just an FYI.

This could be simplified to use only one piece of state like so:
```ts
const [stepIndex, setStepIndex] = useState(0);
const selectedItem = updatedChecklistItems[stepIndex];
```

Reply:
This comment keeps getting marked as resolved, but the change I'm suggesting seems to be getting overlooked. Any reason?

Reply:
Sorry, I thought I had addressed it. Should be better now.

Reply:
Hm. It's still the same 😄

Reply:
Ah, I think I was seeing a stale version in GitHub. It _has_ changed, although there's now an unused value:

<img width="408" alt="Screen Shot 2023-04-05 at 10 12 40 AM" src="https://user-images.githubusercontent.com/29597/230107686-186c8fcb-bf97-45e4-9402-423a0db41c96.png">

I'll update it !